sdcm/cluster.py

```diff
@@ -2803,9 +2803,16 @@ def is_cqlsh_support_cloud_bundle(self):
 
     @property
     def is_replacement_by_host_id_supported(self):
+        if self.parent_cluster.params.get("use_preinstalled_scylla"):
+            # since we need this logic before starting a node, and in use_preinstalled_scylla case
+            # scylla is not yet installed, we should check other running nodes for version
+            other_up_node = random.choice([n for n in self.parent_cluster.nodes if n != self])
+            scylla_version = other_up_node.scylla_version
```

Comment:
The `up` part is not guaranteed here. It may be down caused by a parallel disruptive nemesis...

Reply:
I don't need scylla up, I need the node to be up, and have scylla installed.
it's gonna be tricker to guarante all that with parallel nemesis.

maybe the target node of the nemesis would be better to test, regardless, and not the new node, or other nodes

Reply:
I've simplified it, and just using the target_node of the nemesis, that node should be installed and ready

Reply:
fixed